pip/pep425tags.py

```diff
@@ -124,7 +124,54 @@ def get_platform():
         split_ver = release.split('.')
         return 'macosx_{0}_{1}_{2}'.format(split_ver[0], split_ver[1], machine)
     # XXX remove distutils dependency
-    return distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    result = distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    if result == "linux_x86_64" and sys.maxsize == 2147483647:
```

Comment:
If you run 32 bit Python on a 64 bit linux (e.g. via anaconda or even if you debootstrap a i686 ubuntu inside a 64 bit ubuntu), then `distutils.util.get_platform()` returns  "linux_x86_64" even though Python itself is 32 bit.

I don't know if this is a bug of distutils or not.